code/game/objects/items.dm

```diff
@@ -566,6 +566,23 @@ obj/item/proc/item_action_slot_check(slot, mob/user)
 		. = 'sound/weapons/sear.ogg'
 	else
 		. = pick('sound/misc/desceration-01.ogg', 'sound/misc/desceration-02.ogg', 'sound/misc/desceration-03.ogg')
+		
+	
+/obj/item/proc/get_knockout_chance()
+	switch(hit_area)

```

Comment:
you need to pass through the H, I and hit_area variables to this proc or get them via some other means.